lib/extensions/stitch_plan_preview.py

```diff
@@ -133,10 +136,14 @@ def rasterize(self, svg: BaseElement, layer: BaseElement, raster_mult: Optional[
                     "export-do"  # Inkscape docs say this should be implicit at the end, but it doesn't seem to be.
                 ]))
 
+                # Extract numbers from returned string. It can include other information such as warnings about the usage of AppImages
+                out = findall(r"\d+\.\d+", out)
```

Comment:
Should this be something like `^\d+\.?\d*$` to make sure it only matches numbers that make up the full line, in case that message contains numbers, and can handle the probably rare case that the coordinates are integers exactly.

... It really sucks that this is apparently something we need to do. This could become a point of brittleness...

Reply:
Yes, it's annoying. You are welcome to add your changes to the branch :)

Reply:
Done. If they work correctly in your testing then IMO it's good to go. :)

Reply:
Great, thanks.
I only removed an unused import and add detection for negative numbers.

Reply:
Good call.